mpt_trie/Cargo.toml

```diff
@@ -14,6 +14,7 @@ exclude = [
 ]
 
 [dependencies]
+alloy = { git = "https://github.com/alloy-rs/alloy" }
```

Comment:
Bringing in `alloy` as a git dependency will prevent us from publishing new releases (`crates.io` rules). So until this gets stabilized we need to wait and leverage some compatibility module as Aatif wrote on `zero-bin`.

Reply:
Ahh ok got it, I will revert this and use the compat in `zero-bin`.